fe/fe-common/pom.xml

```diff
@@ -145,6 +145,12 @@ under the License.
             <groupId>com.esotericsoftware</groupId>
             <artifactId>kryo-shaded</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-collections</groupId>
+            <artifactId>commons-collections</artifactId>
+            <version>${commons-collections.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Since commons-related dependencies are fundamental, it’s best not to categorize them as test dependencies. Doing so could lead to issues when adding new dependencies later on, especially if they have transitive dependencies on commons-collections. Also, it looks like this PR doesn’t actually use commons-collections?